### PR TITLE
Improve UI

### PR DESCRIPTION
**Pull request by ashwin47**
Index page:
Screenshot 2024-07-30 at 7 04 46 PM

Show page:
Screenshot 2024-07-30 at 7 05 28 PM

### CODE REVIEW

File: app/views/mailbin/messages/index.html.erb [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
-  <ul>
-    <% @emails.each do |email| %>
-      <li><%= link_to email.subject, message_path(email.message_id) %> - <%= l email.date, format: :long %></li>
-    <% end %>
-  </ul>
+  <table class="email-list">
+    <thead>
+      <tr>
+        <th>Subject</th>
+        <th>Date</th>
```
Comment (kaka-ruto) [inline app/views/mailbin/messages/index.html.erb:12]: How about adding a Delete link or icon to the far right? So I can delete any email I like on the index page as well, without having to open it

**Comment by excid3** [comment]
Thanks for starting on this. I am not sure, but we may want to redo everything to use Turbo Frames and have a sidebar to view the emails quicker than jumping between pages.

**Comment by kaka-ruto** [comment]
> Thanks for starting on this. I am not sure, but we may want to redo everything to use Turbo Frames and have a sidebar to view the emails quicker than jumping between pages.

That does sound like a better idea. It'll be faster to navigate.

I do think this PR is an improvement though? So we can have it and then have another one that uses Turbo Frames?

**Comment by jarkelen** [comment]
Yes please! This looks like a nice first UI improvement.

**Pull request closed without merging**